app/scripts/controllers/transactions/index.js

```diff
@@ -739,7 +750,7 @@ export default class TransactionController extends EventEmitter {
     };
     // sign tx
     const fromAddress = txParams.from;
-    const common = await this.getCommonConfiguration();
+    const common = await this.getCommonConfiguration(txParams.from);
```

Comment:
nit: use `fromAddress` here?

Reply:
`txParams.from` is used everywhere I believe